robyn/__init__.py

```diff
@@ -51,25 +51,44 @@ def add_route(self, route_type, endpoint, handler):
         """
         number_of_params = len(signature(handler).parameters)
         self.routes.append(
-            (route_type,
-             endpoint,
-             handler,
-             asyncio.iscoroutinefunction(handler), number_of_params)
+            (
+                route_type,
+                endpoint,
+                handler,
+                asyncio.iscoroutinefunction(handler),
+                number_of_params,
+            )
         )
 
-    def add_directory(self, route, directory_path, index_file=None, show_files_listing=False):
+    def add_directory(
+        self, route, directory_path, index_file=None, show_files_listing=False
+    ):
         self.directories.append((route, directory_path, index_file, show_files_listing))
 
     def add_header(self, key, value):
         self.headers.append((key, value))
 
-    def remove_header(self, key):
-        self.server.remove_header(key)
-
     def add_web_socket(self, endpoint, ws):
         self.web_sockets[endpoint] = ws
 
-    def start(self, url="127.0.0.1", port=5000):
+    def _add_event_handler(self, event_type: str, handler):
+        print(f"Add event {event_type} handler")
+        if event_type.lower() not in {"startup", "shutdown"}:
+            return
+
+        is_async = asyncio.iscoroutinefunction(handler)
+        if event_type.lower() == "startup":
```

Comment:
Very minor, but for clarity, I would consider to:
- Extract the "startup"/"shutdown" string either in an Enum (like Class(str, Enum) or as constants
- DIrectly use "startup"/"shutdown" as keys for the dictionary `event_handlers`

Reply:
Thank you. Updated :D